images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -14,4 +14,12 @@ Describe "Kind" {
     It "Kind" {
         "kind version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "DotnetTLS" {
+    It "Tls 1.2 is enabled" {
+        $protocols = [Net.ServicePointManager]::SecurityProtocol
+        $protocolArr = $protocols -split ', '
+        $protocolArr.Contains('Tls12') | Should -BeTrue
```

Comment:
[Net.ServicePointManager]::SecurityProtocol -band "Tls12" | Should -Be Tls12

Reply:
Please ignore mine message, @al-cheb's proposal is better

Reply:
Done, thanks!